docs/user-guide/examples/advanced_examples/advanced_orchestration.md

```diff
@@ -1,10 +1,362 @@
+
+### Merlin Orchestration Execution
+
+In this specification we will be using the
+[helics_cli](https://github.com/GMLC-TDC/helics-cli) to execute each
+co-simulation run since this is a Monte Carlo simulation. This means
+that `helics_cli` will be executed multiple times with different
+`helics_cli` runner files.
+
+#### Co-simulation Reproduction
+
+Management of multiple iterations of the co-simulation can be done by setting the seed as a function of the number of brokers, where there will be one broker for each iteration. In Python 3.X:
+
+```python
+import argparse
+
+parser = argparse.ArgumentParser(description="EV simulator")
+parser.add_argument(
+    "--seed",
+    type=int,
+    default=0,
+    help="The seed that will be used for our random distribution",
+)
+parser.add_argument("--port", type=int, default=-1, help="port of the HELICS broker")
+
+
+args = parser.parse_args()
+np.random.seed(args.seed)
+```
+
+#### `helics_cli` in Merlin
+
+Since we are using the `helics_cli` to manage and execute all the
+federates, we need to create these runner files for `helics_cli`.
+There is a provided python script called `make_samples_merlin.py` that will
+generate the runner file and a csv file that will be used in the
+study step. `helics_cli` will start each of these federates. In the Merlin
+spec, Merlin will be instructed to execute the `helics_cli` with all the
+generated `helics_cli` runner files.
+
+#### Merlin Specification
+
+##### Environment
+
+In the Merlin spec we will instruct Merlin to execute N number of the
+Monte Carlo co-simulations. The number of samples is the number
+specified as the `N_SAMPLES` env variable in the env section of the
+merlin spec.
+
+```yaml
+env:
+  variables:
+    OUTPUT_PATH: ./UQ_EV_Study
+    N_SAMPLES: 10
+```
+
+We set the output directory to UQ_EV_Study, this is where all the
+output files will be stored. Every co-simulation run executed by
+merlin will have it's own subdirectory in `./UQ_EV_Study`.
+
+##### Merlin Step
+
+Remember this step is for Merlin to setup all the files and data it
+needs to execute it's jobs. In the Monte Carlo co-simulation there is
+a python script we created that will generated the `helics_cli` runner
+files that Merlin will use when it executes the `helics_cli`. The
+`make_samples_merlin.py` script will also output a csv file that Merlin will
+use. The csv file contains all the names of the runner files. Merlin
+will go down this list of file names and execute the `helics_cli` for
+each file name.
+
+```yaml
+merlin:
+  samples:
+    generate:
+      cmd: |
+        python3 $(SPECROOT)/make_samples_merlin.py $(N_SAMPLES) $(MERLIN_INFO)
+        cp $(SPECROOT)/EVMsgFed.py $(MERLIN_INFO)
+        cp $(SPECROOT)/EVControllerMsgFed.py $(MERLIN_INFO)
```

Comment:
@allisonmcampbell I'm also not finding a `make_samples_merlin.py` or the `EVMsgFed.py` files in the HELICS-Examples repository. The closest I'm finding is those two python files and a `make_samples.py` file in https://github.com/GMLC-TDC/HELICS/tree/main/docs/user-guide/examples/user_guide_examples/Example_3

Reply:
`make_samples_merlin.py` now lives here:
HELICS-Examples/user_guide_examples/advanced/advanced_orchestration/simple
renamed EVMsgFed --> Battery
renamed EVControllerMsgFed --> Charger